docker/build-project.sh

```diff
@@ -25,6 +25,12 @@ cd $WORKSPACE
 # get git commit ID of the project for using in buildhistory tagging
 CI_GIT_COMMIT=$(git rev-parse HEAD)
 
+# prepare git config
+#NOTE: bitbake does not pass git enviornment to tasks(do_patch),
+# hence we need to setup the git configuration 
+git config --global user.name "$GIT_COMMITTER_NAME"
+git config --global user.email "$GIT_COMMITTER_EMAIL"
+
 # FIXME: undbound variables used without checking:
```

Comment:
If patch in your changeset would have proper information in header, those wouldn't be needed.